.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       - run:
           name: helm-github-pages
           environment:
-            - GITHUB_PAGES_REPO: cetic/helm-charts
+            - GITHUB_PAGES_REPO: evermind/helm-charts
```

Comment:
Hi @tgruenert!

Can you change here to **cetic/helm-charts** repo?

Reply:
done